src/Controls/src/Core/Interactivity/EventTrigger.cs

```diff
@@ -71,7 +70,7 @@ void AttachHandlerTo(BindableObject bindable)
 			try
 			{
 				_eventinfo = bindable.GetType().GetRuntimeEvent(Event);
-				_handlerdelegate = s_handlerinfo.CreateDelegate(_eventinfo.EventHandlerType, this);
+				_handlerdelegate = ((EventHandler)OnEventTriggered).Method.CreateDelegate(_eventinfo.EventHandlerType, this);
```

Comment:
@MichalStrehovsky I can't remember the rules around `CreateDelegate`, is this something which might cause problems in AOT?

Reply:
This is fine for AOT. If we can invoke a method, we can also create a delegate to it with reflection.

Having to create a delegate just so we can get a `MethodInfo` from it is unfortunate. Looks like this would benefit from a C# `methodof` keyword or - since we're unlikely to get `methodof` - https://github.com/dotnet/runtime/issues/94975.